ipn/ipnlocal/expiry_test.go

```diff
@@ -254,7 +254,7 @@ func TestNextPeerExpiry(t *testing.T) {
 	t.Run("ClockSkew", func(t *testing.T) {
 		t.Logf("local time:  %q", now.Format(time.RFC3339))
 		em := newExpiryManager(t.Logf)
-		em.timeNow = func() time.Time { return now }
+		em.clock = tstest.NewClock(tstest.ClockOpts{Start: time.Unix(1675725516, 0)})
```

Comment:
where these three numbers coming from? What's with the `now` variable that was previously being used that's not?